src/app/components/page/page-item/page-item.component.ts

```diff
@@ -27,11 +25,7 @@ export class PageItemComponent implements OnInit {
 
     ngOnInit() {}
 
-    get isMobile$(): Observable<boolean> {
-        return this.mediaObserver.media$.pipe(
-            map(change => {
-                return change.mqAlias === 'xs';
-            })
-        );
+    isMobile(): boolean {
```

Comment:
Same as the previous, check if this works when you resize the screen. (go from full to small to full screen again)